### PR TITLE
Doc for the OAS 3.0 Beta for Phase 1

### PR DESCRIPTION
**Pull request by rddheath**

### CODE REVIEW

File: modules/ROOT/pages/oas3-support.adoc [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+image::apid-duplicate.png[The Download As option]
++
+. Next to the name of the file, click the three dots.
+. Select *Download As*.
+. (Not pictured) In the *Download As* dialog, select *RAMl 1.0*.
+. After downloading the file, import it into Anypoint Studio or into API Manager.
++
+[NOTE]
+====
+If you want to share an OAS 3 specification in OAS 2, download your OAS 3 specification as-is, and then convert it to OAS 2 by using a tool such as APIMatic.
```
Comment (harshakirank) [inline modules/ROOT/pages/oas3-support.adoc]: For conversions between open API specification versions between OAS 3 and OAS 2, you can use 3rd party/partner tools such as [APImatic transformer](https://www.apimatic.io/dashboard?modal=transform)

File: modules/ROOT/pages/oas3-support.adoc [hunk 2]
```diff
@@ -0,0 +1,72 @@
+= Beta Support for OAS 3.0
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+Phase 1 of the support for OAS 3.0 is included in Anypoint Exchange, API Community Manager, and API Designer.
```
Comment (harshakirank) [inline modules/ROOT/pages/oas3-support.adoc]: Phase 1 of the support for OAS 3.0 enables consumption scenarios in the Anypoint platform. It allows users to showcase their existing external APIs in the platform to drive partner and customer adoption. It includes support for products like Anyppint Exchange, API Community Manager, and API Designer.

File: modules/ROOT/pages/oas3-support.adoc [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+. Select either of these options:
+** Select *Import from Exchange* to see a list of the API specifications that are available from the business organization that your user ID belongs to in Anypoint Platform and a list of the API specifications that are published by MuleSoft.
+** Select *Import* to import an API specification from your local filesystem.
+* Develop and edit OAS 3 specfications.
+* View documentation that is written into OAS 3 specifications.
+* Simulate calls to methods that are in OAS 3 specifications.
+* Publish OAS 3 specifications to Anypoint Exchange.
+* Share OAS 3 specifications as RAML specifications with Anypoint Studio and API Manager.
++
+.How to convert OAS 3.0 files to RAML for import into Anypoint Studio or API Manager
```
Comment (harshakirank) [inline modules/ROOT/pages/oas3-support.adoc]: Let me know your opinion on moving all the conversions under a heading, making it easy for users to understand the options for E2E flows. Something like
== Conversions:
Users can convert between OAS3 specs and RAML 
Using the "duplicate as" option in the API Designer and set the converted spec as the root file before you publish. You can import OAS3 spec into Exchange and use the "download as" feature to convert the spec to RAML and upload it to the Exchange.
Note: The conversions between the languages are the best effort, as there are language-specific features that are not directly supported in other languages and vice-versa. For example, RAML concepts like Resourcetypes, traits, and overlays are not available in OAS3.

File: modules/ROOT/pages/oas3-support.adoc (hunk 2 above)
Comment (rddheath) [reply modules/ROOT/pages/oas3-support.adoc]: @harshakirank , do you really want to use "consumption scenarios" in this doc? I have no idea what that means, and I doubt that readers will, either.

Comment (rddheath) [reply modules/ROOT/pages/oas3-support.adoc]: @harshakirank , "Support for products like"? Don't you mean "support for Anypoint Exchange, API Community Manager, and API Designer"?

Comment (rddheath) [reply modules/ROOT/pages/oas3-support.adoc]: @harshakirank , "In the platform"? What about ACM users who are showcasing their APIs in their own community sites?

File: modules/ROOT/pages/oas3-support.adoc (hunk 3 above)
Comment (rddheath) [reply modules/ROOT/pages/oas3-support.adoc:61]: @harshakirank , can't users convert between OAS3 and OAS2?

File: modules/ROOT/pages/oas3-support.adoc (hunk 2 above)
Comment (harshakirank) [reply modules/ROOT/pages/oas3-support.adoc]: I see your point, I was trying to represent a use case as opposed to the products mentioned above. Maybe we should add use cases something like
With the support in the above products:
Users can catalog their existing OAS3 specs in the platform to enable partner and customer collaboration scenarios. Additionally, users can do edits to their specs with the API Designer.

File: modules/ROOT/pages/oas3-support.adoc (hunk 3 above)
Comment (rddheath) [reply modules/ROOT/pages/oas3-support.adoc]: @harshakirank , I don't think this section makes it easier for users to understand E2E flows because the context in which the conversion takes place is removed. Also, it seems as though you are missing the scenario of downloading as OAS3 from APID for importing specs into Studio or API Manager.

**Comment by harshakirank** [comment]
Looks good to me.

**Review by harshakirank — APPROVED** [review]

File: modules/ROOT/pages/oas3-support.adoc [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+If you want to implement or deploy API specifications written in OAS 3, however, during this beta you must first convert them to RAML or OAS 2.
+
+== Beta Features in Anypoint Exchange
+
+* Publish an existing OAS 3.0 API specification.
++
+On the *Publish a new asset* page, select *REST API - OAS* in the *Asset type* field.
+* Share OAS 3.0 API specifications.
+* View documentation that is written into OAS 3.0 API specifications.
+* Publish OAS 3.0 API specifications to Anypoint Community Manager.
```
Comment (mcampo) [inline modules/ROOT/pages/oas3-support.adoc]:
```suggestion
* Publish OAS 3.0 API specifications to API Community Manager.
```
Also, APIs are not published to ACM from Exchange, they are selected in ACM to be included in the community

File: modules/ROOT/pages/oas3-support.adoc [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+* Share OAS 3.0 API specifications as RAML specifications with Anypoint Studio and API Manager.
```
Comment (mcampo) [inline modules/ROOT/pages/oas3-support.adoc]: @harshakirank what is this interaction referring to?

File: modules/ROOT/pages/oas3-support.adoc [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+. In the *Duplicate As* dialog, select *RAMl 1.0*.
+. Set the new duplicate as the root file in the project.
+. Publish the project to Anypoint Exchange.
+====
+* Share OAS 3.0 specifications as RAML specifications with Anypoint Studio and API Manager.
++
+.How to convert OAS 3.0 files to RAML for import into Anypoint Studio or API Manager
+image::apid-duplicate.png[The Download As option]
++
+. Next to the name of the file, click the three dots.
```
Comment (harshakirank) [inline modules/ROOT/pages/oas3-support.adoc]: This flow works, but takes the users out of the platform by downloading the specs. We should recommend users to instead use the "duplicate as" feature, set the new RAML file as root file, and publish to Exchange to use them into Anypoint Studio or API Manager.

Comment (harshakirank) [reply modules/ROOT/pages/oas3-support.adoc]: @rddheath

File: modules/ROOT/pages/oas3-support.adoc (hunk 5 above)
Comment (harshakirank) [reply modules/ROOT/pages/oas3-support.adoc]: Thanks for cathing this @mcampo. 
Can you review the modification below
Users can download the OAS 3 API specifications as RAML specifications for scaffolding and implementation with 3rd party IDEs. 

For usage in Anypoint Studio and API Manager, the recommended flow is Import in API Designer -> duplicate as "RAML" -> Set the Root file to RAML -> Publish to Exchange." Now, this project is available for implementation in Anypoint Studio and available to manage in API Manager.

**Comment by harshakirank** [comment]
@jdeoliveira @sanjuverm @chiragnarangmulesoft for visibility on the OAS3 beta docs, please review for correctness.

File: modules/ROOT/pages/oas3-support.adoc [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+* View documentation that is written into OAS 3.0 API specifications.
+* Publish OAS 3.0 API specifications to Anypoint Community Manager.
+* Share OAS 3.0 API specifications as RAML specifications with Anypoint Studio and API Manager.
++
+.How to convert OAS 3.0 files to RAML for import into Anypoint Studio or API Manager
+image::aex-download.png[The Download option]
++
+. In the top-right corner of Anypoint Exchange, click *Download*.
+. Select *As RAML*.
+. After downloading the file, import it into Anypoint Studio or into API Manager.
```
Comment (chiragnarangmulesoft) [inline modules/ROOT/pages/oas3-support.adoc]: @harshakirank 
Not sure what does it mean to import into API Manager. You need to be really clear on the steps to follow

Comment (harshakirank) [reply modules/ROOT/pages/oas3-support.adoc]: @chiragnarangmulesoft Can you suggest the text or include your docs counterpart to get the clear steps for API Manager part added, along with links to the main docs if that makes it easy. The goal here is to get the product owners to contribute to this doc to drive towards correctness.

File: modules/ROOT/pages/oas3-support.adoc [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+. Set the new duplicate as the root file in the project.
+. Publish the project to Anypoint Exchange.
+====
+* Share OAS 3.0 specifications as RAML specifications with Anypoint Studio and API Manager.
++
+.How to convert OAS 3.0 files to RAML for import into Anypoint Studio or API Manager
+image::apid-duplicate.png[The Download As option]
++
+. Next to the name of the file, click the three dots.
+. Select *Download As*.
```
Comment (harshakirank) [inline modules/ROOT/pages/oas3-support.adoc]: We need to replace "download as" with the "duplicate as" user action to allow users to use from API Manager and Anypoint Studio.

File: modules/ROOT/pages/oas3-support.adoc [hunk 9]
```diff
@@ -0,0 +1,75 @@
+= Beta Support for OAS 3.0
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+Phase 1 of the support for OAS 3.0 is included in Anypoint Exchange, API Community Manager, and API Designer. You can catalog existing OAS 3.0 API specifications to enable partner and customer collaboration scenarios. You can also create and edit API specifications in OAS 3.
```
Comment (hannanelson) [inline modules/ROOT/pages/oas3-support.adoc]: partner- and customer-collaboration

File: modules/ROOT/pages/oas3-support.adoc [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+
+Perhaps the most important aspect of this beta support is that it lets you publish OAS 3.0 specifications to your API community, so that you can share them with partners and customers to help drive their adoption.
```
Comment (hannanelson) [inline modules/ROOT/pages/oas3-support.adoc]: enables you to?

File: modules/ROOT/pages/oas3-support.adoc [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+
+If you want to implement or deploy API specifications written in OAS 3.0, however, during this beta you must first convert them to RAML or OAS 2.
```
Comment (hannanelson) [inline modules/ROOT/pages/oas3-support.adoc]:
```suggestion
If you want to implement or deploy API specifications written in OAS 3.0 during this beta, you must first convert them to RAML or OAS 2.
```

File: modules/ROOT/pages/oas3-support.adoc [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+
+== Beta Features in Anypoint Exchange
+
+* Publish an existing OAS 3.0 API specification.
++
+On the *Publish a new asset* page, select *REST API - OAS* in the *Asset type* field.
+* Share OAS 3.0 API specifications.
+
+[NOTE]
```
Comment (hannanelson) [inline modules/ROOT/pages/oas3-support.adoc:21]: Does this need a + to be indented?

File: modules/ROOT/pages/oas3-support.adoc [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+====
+If you want to share an OAS 3.0 API specification as OAS 2, download your OAS 3.0 API specification from Anypoint Exchange as-is, and then convert it to OAS 2.0 by using a tool such as APIMatic transformer.
+====
+
+* View documentation that is written into OAS 3.0 API specifications.
```
Comment (hannanelson) [inline modules/ROOT/pages/oas3-support.adoc]:
```suggestion
* View documentation that is written in OAS 3.0 API specifications.
```

File: modules/ROOT/pages/oas3-support.adoc [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+* Publish OAS 3.0 API specifications to API Community Manager.
+* Download OAS 3.0 API specifications as RAML API specifications for scaffolding and implementation with 3rd party IDEs.
```
Comment (hannanelson) [inline modules/ROOT/pages/oas3-support.adoc]:
```suggestion
* Download OAS 3.0 API specifications as RAML API specifications for scaffolding and implementation with third-party IDEs.
```

File: modules/ROOT/pages/oas3-support.adoc [hunk 15]
```diff
@@ -0,0 +1,75 @@
...
+. Click the three dots next to the filename in the *Files* pane and select *Duplicate*.
+. In the *Duplicate As* dialog, select *RAML*.
+. Set the root file of the project to the RAML version of the API specification.
+. Publish the project to Anypoint Exchange.
+
+
+== Beta Features in API Community Manager
+
+* Publish OAS 3.0 specifications to your API community, sharing them with partners and customers to help drive their adoption.
+* View documentation that is written into OAS 3.0 specifications.
```
Comment (hannanelson) [inline modules/ROOT/pages/oas3-support.adoc:42]: I guess this is standard terminology? Ignore previous suggestion.

File: modules/ROOT/pages/oas3-support.adoc [hunk 16]
```diff
@@ -0,0 +1,75 @@
...
++
+When creating a new API-specification project, select *OAS 3.0 (Beta)* in the *Language* field.
+* Import OAS 3.0 specifications from Anypoint Exchange or from your file system.
++
+.How to import files into an API-specification project in API Designer
+image::apid-import.png[Options for importing]
++
+. Click the gear icon in the top-right corner of the text editor.
+. Select either of these options:
+** Select *Import from Exchange* to see a list of the API specifications that are available from the business organization that your user ID belongs to in Anypoint Platform and a list of the API specifications that are published by MuleSoft.
```
Comment (hannanelson) [inline modules/ROOT/pages/oas3-support.adoc]: hard to parse

File: modules/ROOT/pages/oas3-support.adoc [hunk 17]
```diff
@@ -0,0 +1,75 @@
...
+** Select *Import* to import an API specification from your local filesystem.
```
Comment (hannanelson) [inline modules/ROOT/pages/oas3-support.adoc:59]: previously "file system"

File: modules/ROOT/pages/oas3-support.adoc [hunk 18]
```diff
@@ -0,0 +1,75 @@
...
+* Develop and edit OAS 3.0 specfications.
+* View documentation that is written into OAS 3.0 API specifications.
+* Simulate calls to methods that are in OAS 3.0 API specifications.
+* Publish OAS 3.0 API specifications to Anypoint Exchange.
+* Share OAS 3.0 specifications as RAML specifications for scaffolding and implementation with 3rd party IDEs.
```
Comment (hannanelson) [inline modules/ROOT/pages/oas3-support.adoc]: see prev

**Review by hannanelson — APPROVED** [review]
